backend/tracim_backend/views/core_api/user_controller.py

```diff
@@ -76,6 +79,46 @@ def user(self, context, request: TracimRequest, hapic_data=None):
         )
         return uapi.get_user_with_context(request.candidate_user)
 
+    @hapic.with_api_doc(tags=[SWAGGER_TAG__USER_ENDPOINTS])
+    @require_profile(Group.TIM_ADMIN)
+    @hapic.output_body(UserDigestSchema(many=True))
+    def users(self, context, request: TracimRequest, hapic_data=None):
+        """
+        Get all users
+        """
+        app_config = request.registry.settings['CFG']
+        uapi = UserApi(
+            current_user=request.current_user,  # User
+            session=request.dbsession,
+            config=app_config,
+        )
+        users = uapi.get_all()
+        context_users = [
+            uapi.get_user_with_context(user) for user in users
+        ]
+        return context_users
```

Comment:
Reading code i see `uapi.get_all()` return all db users. There is no filter on "current user" to prevent anybody to retrieve complete list of tracim instance users ? Maybe i missed it ?

Reply:
This endpoint is admin only (see `@require_profile(Group.TIM_ADMIN)`) .
Known_members is the endpoint to use for any user.

Reply:
Okay XD